spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java

```diff
@@ -49,8 +49,8 @@
 /**
  * A procedure that creates a view for changed rows.
  *
- * <p>The procedure removes the carry-over rows by default. If you want to keep them, you can set
- * "remove_carryovers" to be false in the options.
+ * <p>The procedure always removes the carry-over rows. Please query {@link SparkChangelogTable}
+ * instead for the use cases doesn't remove carry-over rows.
```

Comment:
`Please query {@link SparkChangelogTable} instead when carry-over rows are not required`

Reply:
> rows are not required

I guess you mean "rows are required".  I have updated.